### PR TITLE
Fix Latitude and Longitudes swap

### LINKED ISSUE #16326: Latitude and Longitudes swap for geometries when SRID is present in edit mode
### Describe the bug

Latitude and Longitudes switch for geometries when SRID is present

### To Reproduce

1. Go to `workd_borders` table
2. Visualize the GIS data to visit see any country
3. Notice it's latitudes and longitudes
4. Now go to edit menu to edit the geometry of the country.
5. The Latitudes and Longitudes are swapped and the country would be in a different place (if the latitude and longitude range is still valid.Eg: Aruba)

### Expected behavior

Latitude and Longitudes should be consistent

### Screenshots

Screenshot from 2020-08-31 00-13-28
Screenshot from 2020-08-31 00-14-26

### Server configuration

-Operating system: Ubuntu 20.04
-Web server: Apache
-Database version: libmysql - mysqlnd 7.4.3
-PHP version: 7.4.3
-phpMyAdmin version: 5.0, master

### PR DESCRIPTION
**Pull request by shucon**
### Description

Fixes #16326

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/Util.php [hunk 1]
```diff
@@ -1695,7 +1695,7 @@ public static function asWKT($data, $includeSRID = false)
             $spatialAsText = 'ST_ASTEXT';
             $spatialSrid = 'ST_SRID';
         }
-        $wktsql     = 'SELECT ' . $spatialAsText . "(x'" . $hex . "')";
+        $wktsql     = 'SELECT ' . $spatialAsText . "(x'" . $hex . "', 'axis-order=long-lat')";
```
Comment (williamdes) [inline libraries/classes/Util.php:1698]: Please refer to the other example in the code base to make this conditional :)

**Review by williamdes — CHANGES_REQUESTED** [review]
This needs to be downbased for 5.0

**Comment by shucon** [comment]
Helpful for changing base branch : https://stackoverflow.com/questions/10853935/change-branch-base

**Review by williamdes — APPROVED** [review]
LGTM

Could you rebase (and squash) and add a test case to the test I added in 0f0dc09dee315a7ac494126dd89e89640ffd2eb5 please ?

**Comment by williamdes** [comment]
Let me know so I can merge this one today if you intend to add a test case :)

**Review by williamdes — APPROVED** [review]
LGTM
Good catch on the phpdoc

**Review by williamdes — APPROVED** [review]

**Pull request merged**